libmscore/beam.cpp

```diff
@@ -2392,7 +2392,7 @@ bool Beam::setProperty(P_ID propertyId, const QVariant& v)
 QVariant Beam::propertyDefault(P_ID id) const
       {
       switch (id) {
-            case P_ID::STEM_DIRECTION: return Direction(Direction::AUTO);
+            case P_ID::STEM_DIRECTION: return Direction::AUTO;
```

Comment:
this is wrong, Direction::AUTO has not type Direction and must be casted. Direction is automatically converted to QVariant.